### PR TITLE
Show Create button on StoreFileDownloadViewer in editor mode.

### PR DESCRIPTION
**Pull request by ggallien13**
Add props to hide create protected file and folder meta options.

### CODE REVIEW

File: packages/react-mui-store-file-viewer/src/components/StoreFileViewer/index.tsx [hunk 1]
```diff
@@ -51,7 +53,17 @@ export const StoreDownloadFileViewer: FC<StoreFileViewerProps> = ({
       defaultSortBy={defaultSortBy}
       defaultShareLinkTitle={defaultShareLinkTitle}
     >
-      <StoreDownloadDataGrid {...props} />
+      <>
+        {configurations?.controls?.editorMode && (
```
Comment (cj-f) [inline packages/react-mui-store-file-viewer/src/components/StoreFileViewer/index.tsx:57]: I wonder if maybe then a separate mode prop, it should just use if apiRef?.upload or apiRef?.folderCreate are provided, and only show the folder button if folderCreate is available

File: packages/react-mui-store-file-viewer/src/components/CreateNewFolderButton/fields.tsx [hunk 2]
```diff
@@ -78,6 +82,7 @@ export const getCreateFolderFields = ({
     label: "Advanced Settings",
     collaspable: true,
     defaultOpen: false,
+    isHidden: () => hideFolderOptions,
```
Comment (cj-f) [inline packages/react-mui-store-file-viewer/src/components/CreateNewFolderButton/fields.tsx:85]: Why are the advanced settings hidden>

Comment (ggallien13) [reply packages/react-mui-store-file-viewer/src/components/CreateNewFolderButton/fields.tsx:85]: This was from Bruce, he wanted a simple create folder modal.

Comment (cj-f) [reply packages/react-mui-store-file-viewer/src/components/CreateNewFolderButton/fields.tsx:85]: Okay

File: packages/react-mui-store-file-viewer/src/components/StoreFileViewer/index.tsx (hunk 1 above)
Comment (ggallien13) [reply packages/react-mui-store-file-viewer/src/components/StoreFileViewer/index.tsx]: Good idea, updated to use apiRef.folderCreate

**Review by cj-f — APPROVED** [review]

**Pull request merged**